src/components/FindUser/FindUser.js

```diff
@@ -11,41 +11,41 @@ import { createNewDialog, onChangeCurrentDialog } from "../../redux/dialogsData/
 import { toggleElementVisibility } from "../../redux/displayMenu";
 import { composeValidators, mustBePhoneNumber } from "../../common/Validators";
 
-class FindUser extends React.Component {
-    addNewDialog = (interlocutorId) => {
-        this.props.toggleElementVisibility(false, false, false)
-        const hasDialog = this.props.dialogs.find(d => d.members.find(m => m.id === interlocutorId) ? d.dialogId : null)
+const FindUser = (props) => {
+    const addNewDialog = (interlocutorId) => {
+        props.toggleElementVisibility(false, false, false)
+        const hasDialog = props.dialogs.find(d => d.members.find(m => m.id === interlocutorId) ? d.dialogId : null)
         if (hasDialog) {
-            this.props.onChangeCurrentDialog(hasDialog.dialogId)
-            this.props.history.push(`/${ hasDialog.dialogId }`)
+            props.onChangeCurrentDialog(hasDialog.dialogId)
+            props.history.push(`/${ hasDialog.dialogId }`)
         } else {
             const dialogId = uuidv4()
-            this.props.createNewDialog(dialogId, this.props.currentUser.id, interlocutorId)
+            props.createNewDialog(dialogId, props.currentUser.id, interlocutorId)
         }
     }
 
-    onChangeField = (value) => {
+    const onChangeField = (value) => {
         const VALIDLENGTH = 7
         if (value && value.length > VALIDLENGTH) {
-            this.props.searchUsers(value)
+            props.searchUsers(value)
         } else {
-            this.props.foundUsers.length && this.props.addFoundUsers([])
+            props.foundUsers.length && props.addFoundUsers([])
         }
     }
 
-    getUserItems = () => {
-        if (this.props.foundUsers.length) {
-            return this.props.foundUsers.map(item => item.id === this.props.currentUser.id ? '' :
-                <UserItem user={ item } addNewDialog={ this.addNewDialog } key={ item.id } />)
+    const getUserItems = () => {
+        if (props.foundUsers.length) {
+            return props.foundUsers.map(item => item.id === props.currentUser.id ? '' :
```

Comment:
so returned array can look like this ['', <UserItem/>, '', '']? Is it correct? Maybe better will be filter an array?